src/Humanizer/Localisation/Formatters/IFormatter.cs

```diff
@@ -43,6 +43,23 @@ public interface IFormatter
         /// <returns></returns>
         string TimeSpanHumanize(TimeUnit timeUnit, int unit, bool toWords = false);
 
+        /// <summary>
+        /// Returns the age format that converts a humanized TimeSpan string to an age expression.
+        /// For instance, in English that format adds the " old" suffix, so that "40 years" becomes "40 years old".
+        /// </summary>
+        /// <returns>Age format</returns>
+        string TimeSpanHumanize_Age();
```

Comment:
This is a breaking change, so I'm not sure if this would be allowed. I think for the purpose of localization it would be better have it as a separate interface. @clairernovotny what do you think?

Reply:
In this case, I think it's okay to add a new method to the interface--I don't expect many/any custom implementations. 

The primary concern behind breaking changes is that newer versions can be used with code compiled against the old one -- and adding a method to the interface is okay there.